sdk/identity/identity/test/public/node/deviceCodeCredential.spec.ts

```diff
@@ -88,8 +94,11 @@ describe("DeviceCodeCredential", function() {
     } catch (e) {
       error = e;
     }
-    assert.equal(error?.name, "AbortError");
-    assert.equal(error?.message, "Cancellation triggered by the AbortSignal");
+
+    assert.equal(error?.name, "AuthenticationRequiredError");
+    assert.ok(
+      error?.message.match("Caller has cancelled token endpoint polling during device code flow")
```

Comment:
Nit: It is generally not advised to check for the error message especially when it is coming from a package outside of our control and may change any time.

Reply:
I'll remove this line!

Reply:
I added it again once I replaced the error with one of our own. The MSAL error message was pretty useless for us, I believe.